recipes/templates/recipes/recipe/create.html

```diff
@@ -19,18 +19,25 @@ <h2>Steps</h2>
 
     <ol id="step-list">
       {% for form_step in form_steps %}
-        <li>{{ form_step.instruction }}</li>
-        <button class="remove-ingredient" type="button">-</button>
-        <button class="add-ingredient" type="button">+</button>
-        <ul>
-          {% for form_stepingr in form_stepingredients %}
-          <li>
-          {{ form_stepingr.ingredient }}
-          {{ form_stepingr.quantity }}
-          {{ form_stepingr.unit }}
-          </li>
-          {% endfor %}
-        </ul>
+        <li data-step-index="{{ forloop.counter0 }}">
+          {{ form_step.instruction }}
+          <button class="remove-ingredient" type="button">-</button>
+          <button class="add-ingredient" type="button">+</button>
+          <ul class="ingredient-list">
+            {% for form_stepingr in form_stepingredients %}
```

Comment:
nit pick, as it's a localized variable, usually I go shorter for loop vars, e.g. `step`